xml/System.Net.Http/HttpRequestMessage.xml

```diff
@@ -461,8 +461,23 @@
       </ReturnValue>
       <Docs>
         <summary>Gets or sets the HTTP message version.</summary>
-        <value>The HTTP message version. The default in the .NET Framework and earlier versions of .NET Core is 1.1. In .NET Core 2.1 and later, it is 2.0.</value>
-        <remarks>Starting with .NET  Core 2.1, the default value of the `Version` property changed from 1.1 to 2.0.</remarks>
+        <value>The HTTP message version. Based on your version of .NET Framework or .NET Core, the default may be `1.1` or `2.0`.</value>
+        <remarks>
+The default message version differs based on the version of .NET Framework or .NET Core you're using.
```

Comment:
If you do the simplification, then you can add a small sentence about the change in behavior from one to the other back to the old one starting with .NET Core 3.0.